examples/llm/structured_output.py

```diff
@@ -0,0 +1,81 @@
+import json
+
+from pydantic import BaseModel, ValidationError
+
+from dynamiq.nodes.llms import OpenAI
+from dynamiq.nodes.types import InferenceMode
+from dynamiq.prompts import Prompt
+
+
+class Document(BaseModel):
+    title: str
+    abstract: str
+    tags: list[str]
+
+
+def validate_json_response(response: str) -> Document:
+    try:
+        parsed_event = Document.model_validate_json(response)
+        print("Pydantic Model Response:", parsed_event)
+        return parsed_event
+    except ValidationError as e:
+        print(f"Validation Error: {e}")
+        print("Raw Response:", response)
+        return None
+
+
+def run_openai_node(prompt: Prompt, schema: dict, inference_mode: InferenceMode):
+    openai_node = OpenAI(model="gpt-4o-mini")
+    response = openai_node.run(input_data={}, schema=schema, prompt=prompt, inference_mode=inference_mode)
```

Comment:
is schema only supported in run method? not in the init?

Reply:
Yes, only in run
https://github.com/dynamiq-ai/dynamiq/blob/e8f4f77b81ce0f8a92051acca9eed9c67b221aa3/dynamiq/nodes/llms/base.py#L35C1-L54C8

Add this to init? Like inference mode also

Reply:
as far as I remember we planned to add `schema` to Input and Output nodes, but it seems this not related to the LLM schema

Reply:
this is related to a different topic.
@olbychos we also need to support this in the init

Reply:
Okay, I’ll add it and let’s merge afterwards